### PR TITLE
Updated pre-existing version of REDIS Quickstart Dashboard and Config.yml for variables

### PR DESCRIPTION
**Pull request by gsidhwani-nr**
Added Variables for instance and prometheus_server. Removed Facet Instance wherever instance is used.

# Summary

A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. **If it is not ready, please mark the pull request as a draft.**

## Pre merge checklist

- [ x] Did you check you NRQL syntax? - Does it work?
- [ x] Did you include an InstallPlan and Documentation reference?
- [ x] Are all documentation links publically accessible?
- [x] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)? 
- [ x] Did you check your descriptive content for spelling and grammar errors?

### Dashboards 

- [x ] Does the PR contain a screenshot for each of your dashboards?
- [x ] Do your screenshots show data?
- [ x] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

Sample Updated Dashboard with NR Prometheus Agent and Prometheus Server for Prometheus Server Selection.
https://staging.onenr.io/0eqwyWYmmjn

### Alerts

- [x ] Did you check that your alerts actually work?

### CODE REVIEW

File: quickstarts/redis-prometheus/config.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 id: 547a0d7c-e01c-4dab-ada7-c9dc5d2a3a71
 # Name of the quickstart (required)
 # sets the URL name of the quickstart on public I/O
-slug: redis-prometheus
+slug: redis-Prometheus
```
Comment (thezackm) [inline quickstarts/redis-prometheus/config.yml:4]: @gsidhwani-nr - why are we changing this slug?

File: quickstarts/redis-prometheus/config.yml [hunk 2]
```diff
@@ -26,7 +26,8 @@ description: |+
   - Overview Snapshot (# masters, # slaves) + charts with commands/sec and commands/sec by node
   - Charts showing connected clients, connected clients by node, changes since last save by node, expired keys/second by node, memory used by node, and blocked clients.
   - Charts showing keyspace hit ratio by node, evicted keys/second by node, input bytes/second by node, network I/O per second, and output bytes / second by node.
-
+  - This quickstart was built based on `redis (Kubernetes cluster)` metrics sent to New Relic through remote write configurations with Prometheus Agent or Prometheus server.
```
Comment (thezackm) [inline quickstarts/redis-prometheus/config.yml:29]: @gsidhwani-nr - change this to:

```
This quickstart is based on metrics sent to New Relic using Prometheus Agent or Prometheus Server.
```

File: quickstarts/redis-prometheus/config.yml [hunk 3]
```diff
@@ -36,7 +37,7 @@ description: |+
...
 
 summary: |
-  New Relic provides crucial tools for monitoring your Redis instances using the [Prometheus Remote-Write](https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-remote-write/set-your-prometheus-remote-write-integration/) integraton. It supports features such as a visual dashboard complete with charts detailing all critical performance and health metrics relevant to your Redis system.
+  New Relic provides crucial tools for monitoring your Redis instances using the [Prometheus Remote-Write](https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-remote-write/set-your-prometheus-remote-write-integration/) integraton. It supports features such as a visual dashboard complete with charts detailing all critical performance and health metrics relevant to your Redis system. This quickstart was built based on `redis (Kubernetes cluster)` metrics sent to New Relic through remote write configurations with Prometheus Agent or Prometheus server.
```
Comment (thezackm) [inline quickstarts/redis-prometheus/config.yml:40]: @gsidhwani-nr - same as above, change this last sentence to:

```
This quickstart is based on metrics sent to New Relic using [Prometheus Agent or Prometheus Server.
```

**Review by thezackm — CHANGES_REQUESTED** [review]
@gsidhwani-nr - review added for `prometheus-growth` project

File: quickstarts/redis-prometheus/config.yml (hunk 2 above)
Comment (gsidhwani-nr) [reply quickstarts/redis-prometheus/config.yml:29]: Applied suggestion.

File: quickstarts/redis-prometheus/config.yml (hunk 3 above)
Comment (gsidhwani-nr) [reply quickstarts/redis-prometheus/config.yml:40]: Applied Changes

**Review by thezackm — DISMISSED** [review]
approved for `prometheus-growth` project

File: quickstarts/redis-prometheus/config.yml [hunk 4]
```diff
@@ -53,19 +54,25 @@ keywords:
...
 # Allows us to construct reusable "install plans" and just use their ID in the quickstart config
 installPlans:
   - setup-prometheus
+  - setup-prometheus-agent-mode
 
 # Documentation references
 documentation:
+  - name: Prometheus installation docs
+   description: |
+     Use Prometheus remote_write or Prometheus Agent
```
Comment (aswanson-nr) [inline quickstarts/redis-prometheus/config.yml:70]:
```suggestion
  - name: Prometheus installation docs
     description: |
       Use Prometheus remote_write or Prometheus Agent
```

File: quickstarts/redis-prometheus/config.yml [hunk 5]
```diff
@@ -53,19 +54,25 @@ keywords:
...
   - write
   - kubernetes
+  - exporter
   - database
 
 # Reference to install plans located under /install directory
 # Allows us to construct reusable "install plans" and just use their ID in the quickstart config
 installPlans:
   - setup-prometheus
+  - setup-prometheus-agent-mode
```
Comment (aswanson-nr) [inline quickstarts/redis-prometheus/config.yml:64]: This install plan does not exist. Please use an existing one or create a new one!

**Review by aswanson-nr — CHANGES_REQUESTED** [review]
This looks pretty much good to go! Just two things to get cleaned up

File: quickstarts/redis-prometheus/config.yml (hunk 4 above)
Comment (aswanson-nr) [reply quickstarts/redis-prometheus/config.yml:70]: This  should fix the broken yaml linting!

File: quickstarts/redis-prometheus/config.yml [hunk 6]
```diff
@@ -36,7 +37,7 @@ description: |+
...
 
 summary: |
-  New Relic provides crucial tools for monitoring your Redis instances using the [Prometheus Remote-Write](https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-remote-write/set-your-prometheus-remote-write-integration/) integraton. It supports features such as a visual dashboard complete with charts detailing all critical performance and health metrics relevant to your Redis system.
+  New Relic provides crucial tools for monitoring your Redis instances using the [Prometheus Remote-Write](https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-remote-write/set-your-prometheus-remote-write-integration/) integraton. It supports features such as a visual dashboard complete with charts detailing all critical performance and health metrics relevant to your Redis system. This quickstart is based on metrics sent to New Relic using [Prometheus Agent or Prometheus Server.
```
Comment (thezackm) [inline quickstarts/redis-prometheus/config.yml:40]: typo: `[Prometheus` should be `Prometheus`

File: quickstarts/redis-prometheus/config.yml (hunk 5 above)
Comment (thezackm) [reply quickstarts/redis-prometheus/config.yml:66]: pending in https://github.com/newrelic/newrelic-quickstarts/pull/1503

File: dashboards/redis-prometheus/redis-prometheus.json [hunk 7]
```diff
@@ -1,462 +1,569 @@
...
+        {
+          "value" : {
+            "string": "*"
+          }
+        }
+      ],
+      "nrqlQuery": {
+        "accountIds": [
+          0
+        ],
```
Comment (thezackm) [inline dashboards/redis-prometheus/redis-prometheus.json:542]: set to `"accountIds": []`

File: dashboards/redis-prometheus/redis-prometheus.json [hunk 8]
```diff
@@ -1,462 +1,569 @@
...
```
Comment (thezackm) [inline dashboards/redis-prometheus/redis-prometheus.json:562]: set to `"accountIds": []`

**Review by thezackm — CHANGES_REQUESTED** [review]
@gsidhwani-nr - changes requested; ty

File: quickstarts/redis-prometheus/config.yml (hunk 6 above)
Comment (gsidhwani-nr) [reply quickstarts/redis-prometheus/config.yml]: Modified

File: dashboards/redis-prometheus/redis-prometheus.json (hunk 8 above)
Comment (gsidhwani-nr) [reply dashboards/redis-prometheus/redis-prometheus.json]: Modified

**Review by thezackm — DISMISSED** [review]

File: quickstarts/redis-prometheus/config.yml [hunk 9]
```diff
@@ -36,7 +37,7 @@ description: |+
...
 
 summary: |
-  New Relic provides crucial tools for monitoring your Redis instances using the [Prometheus Remote-Write](https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-remote-write/set-your-prometheus-remote-write-integration/) integraton. It supports features such as a visual dashboard complete with charts detailing all critical performance and health metrics relevant to your Redis system.
+  New Relic provides crucial tools for monitoring your Redis instances using the [Prometheus Remote-Write](https://docs.newrelic.com/docs/infrastructure/prometheus-integrations/install-configure-remote-write/set-your-prometheus-remote-write-integration/) integraton. It supports features such as a visual dashboard complete with charts detailing all critical performance and health metrics relevant to your Redis system. This quickstart is based on metrics sent to New Relic using Prometheus Agent or Prometheus Server.
```
Comment (josephgregoryii) [inline quickstarts/redis-prometheus/config.yml:40]: We need to keep the summary under 500 characterts.
Also to note, there is _no_ support for markdown inside our summary fields, so i would recommend removing that from the config

**Review by josephgregoryii — COMMENTED** [review]
Looks great, one more change i found that needs to be addressed; thanks!

File: quickstarts/redis-prometheus/config.yml (hunk 9 above)
Comment (josephgregoryii) [reply quickstarts/redis-prometheus/config.yml:40]: Screen Shot 2022-11-29 at 10 18 05 AM

Comment (josephgregoryii) [inline quickstarts/redis-prometheus/config.yml:40]:
```suggestion
  New Relic provides crucial tools for monitoring your Redis instances using the Prometheus Remote-Write integraton. It supports features such as a visual dashboard complete with charts detailing all critical performance and health metrics relevant to your Redis system. This quickstart is based on metrics sent to New Relic using Prometheus Agent or Prometheus Server.
```

**Review by josephgregoryii — APPROVED** [review]

**Pull request merged**